misc/test_coo_cusolver.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import scipy.io as sio
+
+import taichi as ti
+
+ti.init(arch=ti.cuda)
+
+
+@ti.kernel
+def init_b(b: ti.types.ndarray(), nrows: ti.i32):
+    for i in range(nrows):
+        b[i] = 1.0 + i / nrows
+
+
+@ti.kernel
+def print_x(x: ti.types.ndarray(), ncols: ti.i32, length: ti.i32):
+    for i in range(ncols - length, ncols):
+        print(x[i])
+
+
+print(">> load sparse matrix........")
+A_raw_coo = sio.mmread('misc/lap2D_5pt_n100.mtx')
```

Comment:
The file seems too large. Is it possible to create a smaller one or shall we move it elsewhere?

Reply:
Could I download the file when running this script?  This file is used to show that the results of this solver is identical with [cuSolverSp_LinearSolver](https://github.com/NVIDIA/cuda-samples/blob/master/Samples/4_CUDA_Libraries/cuSolverSp_LinearSolver/cuSolverSp_LinearSolver.cpp).

Reply:
Or, we can generate a matrix randomly here. Just remember to make sure the matrix is inverteable. :-)

Reply:
Downloading is fine :-)